Doc/library/gettext.rst

```diff
@@ -462,9 +462,9 @@ it in ``_('...')`` --- that is, a call to the function :func:`_`.  For example::
 
    filename = 'mylog.txt'
    message = _('writing a log message')
-   fp = open(filename, 'w')
-   fp.write(message)
-   fp.close()
+   with open(filename, 'w') as fp:
+       fp.write(message)
+       fp.close()
```

Comment:
close is now unnecessary :)

Reply:
yep, sure, I have updated the branch with your patch and when there was an issue, I have just changed the content. +1